index.html

```diff
@@ -170,10 +170,11 @@ <h2 class="head">contact me</h2>
         <p class="paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
           I’d love to help with it</p>
              <form action="https://formspree.io/f/xknegber" method="post">
-                <input type="text" placeholder="Alexendra loremipsum" class="name" minlength="30" maxlength="30" required>
-                <input type="email" placeholder="alexendra@gmail.com" class="em" required>
+                <input type="text" placeholder="Enter your full name" class="name" minlength="30" maxlength="30" required>
```

Comment:
- [ ] Kindly change the minimum number of characters for the name input as it is not allowing me to submit if the names has less then 30 characters